portfolio/src/main/webapp/interests.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="UTF-8">
+        <title>Ross Johnson's Interests</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <center>
+            <h1>Ross Johnson's Interests</h1>
+            <img class="interestimg" src="./images/flutter.png"></img>
+            <img class="interestimg" src="./images/tf.jpg"></img>
+            <img class="interestimg" src="./images/opencv.png"></img>
+            <p></p>
```

Comment:
extra element? If this is to add space between, you can either add margin on the element below or above.